src/main.rs

```diff
@@ -373,42 +373,51 @@ fn test_merge_avgs() {
 /// Print a list of `avg::AVG`
 fn print_avgs(options: &Options, avgs: &BTreeMap<String, avg::AVG>) {
     let mut t = term::stdout().expect("term::stdout failed");
-
     for (pkg, avg) in avgs {
-        t.fg(avg.severity.to_color()).expect("term::fg failed");
-
         match avg.fixed {
             Some(ref v) if avg.status != enums::Status::Vulnerable => {
-                if options.quiet >= 2 {
-                    writeln!(t, "{}", pkg).expect("term::writeln failed");
-                } else if options.quiet == 1 {
-                    writeln!(t, "{}>={}", pkg, v).expect("term::writeln failed");
+                // Quiet option
+                if options.quiet != 0 {
```

Comment:
Can you please change this to `>= 1`? That way it will also work if someone specifies the quiet argument three times.